cmd/system-probe/config/config.go

```diff
@@ -144,6 +145,7 @@ func load(configPath string) (*Config, error) {
 		ProfilingURL:         cfg.GetString(key(spNS, "internal_profiling.profile_dd_url")),
 		ProfilingAPIKey:      aconfig.SanitizeAPIKey(cfg.GetString(key(spNS, "internal_profiling.api_key"))),
 		ProfilingEnvironment: cfg.GetString(key(spNS, "internal_profiling.env")),
+		ProfilingPeriod:      cfg.GetInt(key(spNS, "internal_profiling.period")),
```

Comment:
This needs to also be defined in `pkg/config/system_probe.go`, which can also take a default value.

Reply:
+1 

Yup, please let's do this before merging.

Reply:
Done ✅